ocaml/doc/OMakefile

```diff
@@ -23,7 +23,20 @@ sdk-install: doc-html
 	cp *.js *.html *.css *.png $(SDK)/docs/html/xenserver/
 	cp xenserver/* $(SDK)/docs/html/xenserver/
 
-noarch-install: sdk-install
+.PHONY: doc-html-install
+doc-html-install: doc-html
+	mkdir -p $(DESTDIR)$(DOCDIR)/html/xcp
+	sh -c 'cp -r -L api $(DESTDIR)$(DOCDIR)/html/xcp/'
```

Comment:
I've just run a build on this branch and the OMakefile does not recognise -L when used outside the `sh`.